iceoryx2-cli/lib/src/output.rs

```diff
@@ -23,25 +23,29 @@ use iceoryx2::service::ServiceDetails as IceoryxServiceDetails;
 use iceoryx2::service::ServiceDynamicDetails as IceoryxServiceDynamicDetails;
 use iceoryx2_pal_posix::posix::pid_t;
 
-#[derive(serde::Serialize, Eq, PartialEq, Ord, PartialOrd)]
+#[derive(serde::Serialize, Debug, Eq, PartialEq, Ord, PartialOrd)]
 pub enum ServiceDescriptor {
     PublishSubscribe(String),
```

Comment:
Wouldn't here `ServiceName` a better fit than `String`?

Reply:
No because this is converted directly to the output format (e.g. JSON)